packages/patternfly-4/react-styles/src/build/jest/snapshot-serializer/cssUtils.js

```diff
@@ -39,16 +58,19 @@ function getComputedStyles(classNames, cssStr) {
   );
   const cs = dom.window.getComputedStyle(dom.window.document.body.children[0]);
   const values = {};
-  for (let i = 0; i < cs.length; i++) {
-    const key = cs[i];
+  for (const key of cs.length) {
     const value = cs
       .getPropertyValue(key)
-      .replace(/rgb\(([\d|,|\s]+)\)/g, (full, match) => rgbToHex(...match.split(',').map(n => parseInt(n, 10))));
+      .replace(/rgb\(([\d|,|\s]+)\)/g, (full, match) => rgbToHex(...match.split(',').map(n => parseInt(n))));
```

Comment:
We should be assigning the Radix '10' to the parseInt function. Not doing this can give us unreliable results from the function call.

Reply:
Added radix 10 back in